R/run.R

```diff
@@ -617,11 +640,15 @@ run_script <- function(lines, cmd = find_r(), ..., metadata, progress_bar, read_
         output = NULL,
         rscript = lines
       ),
+      machine_info=NULL,
       context = metadata$context,
       github = metadata$github
     )
   }
 
+  result <- augment_result(result)
```

Comment:
We could drop this. It make the results passed back to labs/benchmarks more fully-featured, but it repopulates most of that anyway; when arrowbench is called through abci directly, it will use the higher-level benchconnect run workflow.

Reply:
Reverted; there are just not enough safeguards at this point in `run_one()` that `run_id` etc. are set correctly. This still works through the higher-level interface.